Pollo/ViewControllers/PollsDateViewController.swift

```diff
@@ -88,40 +121,57 @@ class PollsDateViewController: UIViewController {
+            exportNoticeView.layer.borderColor = UIColor.white.cgColor
+            view.addSubview(exportNoticeView)
+        }
+        
         let updater = ListAdapterUpdater()
         adapter = ListAdapter(updater: updater, viewController: self)
         adapter.collectionView = collectionView
         adapter.dataSource = self
         
+        BannerController.shared.show(NotificationBanner.connectingBanner())
+    }
+    
+    func setupConstraints() {
         collectionView.snp.makeConstraints { make in
             make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(collectionViewTopPadding)
             make.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom)
             make.width.equalToSuperview()
             make.centerX.equalToSuperview()
         }
-        BannerController.shared.show(NotificationBanner.connectingBanner())
-    }
-    
-    func setupNavBar() {
-        navigationController?.setNavigationBarHidden(false, animated: false)
-        navigationController?.navigationBar.barStyle = .black
-        // REMOVE BOTTOM SHADOW
-        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
-        navigationController?.navigationBar.shadowImage = UIImage()
-        let textHeight = navigationController?.navigationBar.frame.height ?? 0
-        navigationTitleView = getNavigationTitleView(primaryText: session.name, primaryTextHeight: textHeight, secondaryText: "Code: \(session.code)", secondaryTextHeight: textHeight, userRole: userRole, delegate: self)
-        self.navigationItem.titleView = navigationTitleView
-        
-        let backImage = UIImage(named: "back")?.withRenderingMode(.alwaysOriginal)
-        self.navigationItem.leftBarButtonItem = UIBarButtonItem(image: backImage, style: .done, target: self, action: #selector(goBack))
         
         if userRole == .admin {
-            createPollButton = UIButton()
-            createPollButton.isHidden = pollsDateArray.last?.polls.last?.state == .live
-            createPollButton.setImage(#imageLiteral(resourceName: "whiteCreatePoll"), for: .normal)
-            createPollButton.addTarget(self, action: #selector(createPollBtnPressed), for: .touchUpInside)
-            let createPollBarButton = UIBarButtonItem(customView: createPollButton)
-            self.navigationItem.rightBarButtonItems = [createPollBarButton]
+            exportNoticeView.snp.makeConstraints { make in
+                make.bottom.equalTo(view.snp.bottom).inset(exportNoticeViewBottomPadding)
+                make.height.equalTo(exportNoticeViewHeight)
+                make.width.equalTo(exportNoticeViewWidth)
+                make.centerX.equalToSuperview()
+            }
+            
+            exportNoticeLabel.snp.makeConstraints { make in
+                make.top.equalTo(exportNoticeView.snp.top).offset(exportNoticeLabelTopPadding)
+                make.height.equalTo(exportNoticeLabelHeight)
```

Comment:
Delete the height constraint and constant, add bottom constraint instead from superview, then add centerY constraint

Reply:
actually nvm you don't need centerY